### PR TITLE
DEVELOPER-2403 Re-brand of Product Pages

### PR DESCRIPTION
**Pull request by adelasofia**

### CODE REVIEW

**Comment by paulrobinson** [comment]
Looks great! 

@insectengine can you review the visuals? 

@wesbos can you review the code and cherry pick this into your rebrand branch?

@adelasofia It's probably worth doing future changes on top of Wes' rebrand branch so as to reduce the risk of merge conflicts.

**Comment by insectengine** [comment]
@paulrobinson 

Are there links to a viewable staging site for these?

**Comment by adelasofia** [comment]
@insectengine here you go http://it-developers-pr.stage.redhat.com/pr/697/build/463/#! :smiley:

**Comment by ollyjshaw** [comment]
retest this please

File: _layouts/product-base.html.slim [hunk 1]
```diff
@@ -7,42 +7,43 @@ secondary_section: supported-products
...
-          i.fa.fa-caret-right
-      - if page.product.has_community_page
-        a.large-24.medium-12.columns(href="../community/")
-          strong Make it better
+section(class="product-graphics #{page.product.product_category}")
+  #hero-try-or-buy(class="row #{page.product.product_category}")
+    .large-24.columns.hello-world
+      .small-24.large-6.columns
+        h2 Hello World!
+        p BUILD SOMETHING TODAY
```
Comment (wesbos) [inline _layouts/product-base.html.slim]: This should be `Build Something Today` and then use CSS `text-transform:uppercase;` to make caps. It's a small thing, but helps with accessibility and readability

**Comment by wesbos** [comment]
Other than my comment :+1:

**Comment by adelasofia** [comment]
Fixed, thanks @wesbos!

**Comment by adelasofia** [comment]
@paulrobinson Apologies, I didn't realize DEVELOPER-2022 should also go in this PR, I just added the icons

**Comment by paulrobinson** [comment]
@wesbos please can you cherry-pick this into your rebrand branch. Closing the PR.

**Comment by wesbos** [comment]
Looks like there are a bunch of conflicts on here because I've already done the product page headers. you might just need to pull over the pieces you need manually rather than try resolve this

**Pull request closed without merging**